src/Stereo.h

```diff
@@ -40,6 +41,8 @@ class StereoSGBM: public node::ObjectWrap {
     JSFUNC(Compute);
 };
 
+#if CV_VERSION_MAJOR < 3
```

Comment:
As per http://docs.opencv.org/master/db/dfa/tutorial_transition_guide.html#gsc.tab=0 we should be using CV_MAJOR_VERSION instead of CV_VERSION_MAJOR if my understanding is correct.